apps/ts-minbar-test-react-components/src/index.ts

```diff
@@ -0,0 +1,58 @@
+import config from '@fluentui/scripts/config';
+import sh from '@fluentui/scripts/gulp/sh';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import {
+  addResolutionPathsForProjectPackages,
+  packProjectPackages,
+} from '@fluentui/scripts/projects-test/packPackages';
+import { createTempDir, log } from '@fluentui/scripts/projects-test/utils';
+
+async function performTest() {
+  try {
+    const logger = log('test:ts-minbar-react-components:typings');
```

Comment:
nit, I don't think you need the :typings suffix here since this package is only for testing typings